PI-System-Deployment-Tests/source/OSIsoft.PISystemDeploymentTests.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="packages\xunit.runner.visualstudio.2.4.2\build\net452\xunit.runner.visualstudio.props" Condition="Exists('packages\xunit.runner.visualstudio.2.4.2\build\net452\xunit.runner.visualstudio.props')" />
```

Comment:
You know, I tried updating to this version and it caused issues in all our build pipelines. If this works for them, I may need to re-investigate those issues because I assumed it was a problem with that version.